js/test/root/teststrings.js

```diff
@@ -3244,9 +3270,10 @@ module.exports.teststrings = {
         test.ok(str !== null);
 
         if (ilib._getPlatform() === "nodejs") {
-            var version = process.versions["node"];
-            var majorVersion = version.split(".")[0];
-            if (Number(majorVersion) < 16) {
+            var cldrVersion = process.versions["cldr"];
+            if (Number(cldrVersion) < 36) { // Intl.PluralRules doesn't support this locale until this version.
+                test.equal(str.formatChoice(1000000), "The items are many");
+            } else if (Number(cldrVersion) >= 36 && Number(cldrVersion) < 40 ) {
```

Comment:
Should that be `Number(cldrVersion) <= 40`

Reply:
It changed to version 40. https://unicode-org.github.io/cldr-staging/charts/40/supplemental/language_plural_rules.html#pt